cves/kernel/CVE-2013-0290.yml

```
@@ -235,8 +244,10 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
+  - commit: 3f518bf745cbd6007d8069100fb9cb09e960c872
+    note: |
+      Interesting the initial commit which created the issue was made almost 
+      exactly a year before the vulnerability was fixed
```

I chose not to put my vulnerability-contributing commits as interesting commits, but I think this is fine.

I would definitely reword that note. It's hard to understand. Consider:
```suggestion
  commits:
  - commit: 3f518bf745cbd6007d8069100fb9cb09e960c872
    note: |
      The commit that first introduced the issue. It was made approximately one
      year prior to the discovery and resolution of the vulnerability.
```